client/tasks/task-list.tsx

```diff
@@ -0,0 +1,174 @@
+/**
+ * External dependencies
+ */
+import { __, _n, sprintf } from '@wordpress/i18n';
+import { useEffect, useRef, useState } from '@wordpress/element';
+import { Card, CardBody, CardHeader } from '@wordpress/components';
```

Comment:
It looks like the `CardBody` import here isn't used.

Reply:
Good catch 👍